index.js

```diff
@@ -67,6 +67,54 @@ function toggleColor(element) {
     }
 }
 
+const headerHText = document.querySelector('h1#header')
+
+headerHText.addEventListener('click', () => {
+    toggleColor(headerHText)
+})
+
 /***** Deliverable 2 *****/
+const profileLikeButton = document.querySelector("#profile > button.like-button")
+
+function addLike() {
+    let likeTextLine = document.querySelector("#profile > p.likes")
+    // let numberOfLikes = parseInt((likeTextLine.textContent).split(" ")[0])
+    // let likeWord = (likeTextLine.textContent).split(" ")[1]
+
+    // numberOfLikes ++
+    // newTextLine = `${numberOfLikes} ${likeWord}`
+
+    // likeTextLine.textContent = newTextLine
+    traveler.likes ++
+    likeTextLine.textContent = `${traveler.likes} Likes`
+}
+
+profileLikeButton.addEventListener("click", addLike)
 
 /***** Deliverable 3 *****/
+
+//https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcRScyAlwz8XmCGVAelZEgA6BiWzL6nFZfmhUw&usqp=CAU
+//https://www.youtube.com/watch?v=xauuxEwHFxw
+//Guinea pigs galore! Not for eating!
+
+const sightingSubmitButton = document.querySelector("#new-animal-sighting-form > input[type=submit]")
+sightingSubmitButton.addEventListener("click", (event) => event.preventDefault())
```

Comment:
Okay rad so you got this working butttt you're making more work for yourself! If we just define the `<form>` then we can just listen for a submit event on the entire form! That also makes extracting the data you get from the form muchhh easier.